config/tips.txt

```diff
@@ -0,0 +1,136 @@
+Enabling the neural lace option will allow you to be cloned, instead of being transferred to a synthetic body or borged.
+Pirate vessels come in sizes and armament. It's good to consider how well-armed you are for a fight.
+Pirate vessels usually contain a vault full of materials and loot, it's always worthwhile boarding them.
+Critically low blood-levels will cause brain damage. Iron + Sugar pills, IV drips with blood and eating food will help you restore blood levels.
+Lasers will cause burn damage and remove blood in your body.
+Exposure to low air environments will cause your lungs to collapse, always wear internals in low air environments to combat this.
+Not all exo-planets will have a breathable atmosphere, or safe temperatures. Shuttles will often have a sensor console that will have data to reflect the atmosphere.
+Some exo-planets may have flammable atmospheres, laser weaponry, smoking and gun-fire may set areas, or the entire planet on fire.
+Long-Range Holocommunicators allow you to talk to other vessels in the sector, provided someone is around on the other end to reply.
+Short-wave radios allow you to communicate with each-other in areas without telecommunications.
+Abandoned and derelict ruins may hold ancient treasures, or horrors.
+No two alien anomalies will have the same activation state, use the anomaly computer to analyze and outline what you may need to do to activate them.
+It is considered poor taste to decapitate someone and keep their neural lace, always give people a chance to come back into the round in some way.
+Straightjackets and Electropacks should never be used without admin approval, as they are impossible to get out of.
+Stowaways have a chance to be spawnable after the round starts, however as a Stowaway you won't be defined on the crew manifest or have any ID.
+Most ship AI will start with the manifest lawset, making it follow orders of crew on the ship manifest, as defined by the Captain.
+Ionispheric anomalies may cause AI's to gain new laws, making them act in unusual or fun manners.
+You can use Intellicards to card AI's into a portable state, allowing them to be repaired and have their lawsets visible.
+Rule Lawyering as an AI or Borg with lawsets or custom lawsets defined by antagonists is not fun for anyone.
+Rule Lawering as a imprint implanted person is not fun for anyone, especially the antagonist who used it on you.
+When you are cloned, you forget the last 5 minutes of the events leading up to your death.
+Find a bug, want to give feedback or a feature suggestion? Go to our Github Page - https://github.com/UristMcStation/UristMcStation/issues and write it up!
+The best antagonist rounds usually tend to include most players in someway.
+Losing is !FUN!, each round lost is a lesson to learn and apply for the next round.
+Changelings can take peoples voices, appearance and memories.
+Bluespace Revenants will slowly distort overtime without creating runes and feeding on blood.
```

Comment:
One for @scrdest , but I _believe_ you simply need to quell whatever 'hunger' you're given. It's not necessarily always runes/blood?
```suggestion
Bluespace Revenants will slowly distort over time without creating runes and feeding on blood.
```